frontend/test/metabase/scenarios/organization/timelines-collection.cy.spec.js

```diff
@@ -91,7 +92,11 @@ describe("scenarios > organization > timelines > collection", () => {
       cy.findByText("Add an event").click();
 
       cy.findByLabelText("Event name").type("RC1");
-      cy.findByRole("button", { name: "calendar icon" }).click();
+
+      // New event modal
+      modal().within(() => {
+        cy.findByRole("button", { name: "calendar icon" }).click();
```

Comment:
With changes within this PR, this command will find 2 calendar buttons.
1. Inside the new event modal
2. On the collection page (see the screenshot)
![image](https://user-images.githubusercontent.com/1937582/178522785-68e6ff2e-8e37-4ebe-b1c5-ee826e1401d3.png)